javatests/arcs/android/host/AndroidManifest.xml

```diff
@@ -0,0 +1,19 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="arcs.android.host">
+
+  <uses-sdk android:minSdkVersion="28" android:targetSdkVersion="28"/>
```

Comment:
version 29

Reply:
This doesn't work for some reason and fails travis. This is set to 28 in arcs/android/common/AndroidManifest.xml